les/costtracker.go

```diff
@@ -45,6 +45,7 @@ var (
 		SendTxV2Msg:            {0, 450000},
 		GetTxStatusMsg:         {0, 250000},
 		GetEtherbaseMsg:        {10000, 1},
+		GetGatewayFeeMsg:       {10000, 1},
```

Comment:
A quick look at the definition of this struct makes me think we should change this. It seems much cheap
er than the other overations here, and should have `0` base cost because it costs nothing to initialy get this info
. I would suggest something like `{0, 10000}`.